src/memjs/memjs.ts

```diff
@@ -369,24 +377,25 @@ class Client<Value, Extras> {
               // This ensures the handler will be deleted from the responseCallbacks map in server.js
               // This isn't technically needed here because the logic in server.js also checks if totalBodyLength === 0, but our unittests aren't great about setting that field, and also this makes it more explicit
               handle.quiet = false;
-              callback(null, responseMap, deserialized.extras);
+              callback(null, responseMap);
             } else {
               const key = response.key.toString();
-              responseMap[key] = deserialized.value;
+              responseMap[key] = { ...deserialized, cas: response.header.cas };
             }
           }
           break;
         case constants.RESPONSE_STATUS_KEY_NOT_FOUND:
           if (callback) {
-            callback(null, null, null);
+            // @blackmad: IS THIS CORRECT???
+            callback(null, null);
           }
           break;
```

Comment:
maybe this should just throw? since we're calling GETKQ it should never trigger.

(it's not a bug since the last response is always the no-op that's handled above)